bin/tag-release-continue.js

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env node
+const commander = require("commander");
+const { extend } = require("lodash");
+const api = require("../src/index.js");
+const workflow = require("./workflows/continue");
+const { promoteContinue } = require("./workflows/promote");
+const { prContinue } = require("./workflows/pr");
+const sequence = require("when/sequence");
+
+commander
```

Comment:
This chunk of code is repeated in every `bin` command. Maybe pull it out and reuse it? Something like...

```
const { applyGlobalOptions } = require("../src/utils.js");
applyGlobalOptions( commander );
```